03_Modules/Transactions/src/module/services/transaction.event.service.ts

```diff
@@ -0,0 +1,183 @@
+import { inject, singleton } from "tsyringe";
+import {
+  AdditionalInfoType,
+  AuthorizationStatusEnumType,
+  HandlerProperties,
+  IdTokenInfoType,
+  IMessage,
+  IMessageConfirmation,
+  LoggerService,
+  OcppRequest,
+  OcppResponse,
+  TransactionEventEnumType,
+  TransactionEventRequest,
+  TransactionEventResponse,
+} from "@citrineos/base";
+import {
+  AuthorizationRepository,
+  TransactionEventRepository,
+} from "@citrineos/data";
+import { RabbitMqSender } from "@citrineos/util";
+
+@singleton()
+export class TransactionEventService {
+  constructor(
+    @inject(LoggerService) private readonly loggerService?: LoggerService,
```

Comment:
You can see that the new service automatically gets the needed logger, sender and repositories injected